documentation/Troubleshooting.md

```diff
@@ -1086,162 +1479,249 @@ d????????? ? ? ? ?            ? ..
 ```
+Additionally, disk space usage is increasing, and pods are being evicted due to DiskPressure.
 
-Also, the disk space usage is increasing, and pods are being evicted due to DiskPressure.
+### Alerts
+Not applicable.
 
-**Root cause**: Kubernetes garbage collector cleans up only unused images and containers which are located under `/var/lib/docker`. It starts cleaning up when the disk usage is equal or above `image-gc-high-threshold` (The default value is 85%).
-The pods' eviction due to DiskPressure starts when the free disk space is less than `imagefs.available` (The default value is 15%).
-If other files except images and containers use the disk so that GC cannot free enough space, such an error may happen.
+### Stack trace(s)
+Not applicable.
 
-**Solution**: Move /var/lib/docker to a separate disk of reasonable size. Also setting `image-gc-high-threshold` to a value lower than 100-`imagefs.available` may help.
+### How to solve
+The root cause of this issue is that the Kubernetes garbage collector only cleans up unused images and containers located under `/var/lib/docker`. It initiates cleanup when disk usage reaches the `image-gc-high-threshold` (default is 85%). Pods are evicted due to DiskPressure when the free disk space is less than `imagefs.available` (default is 15%).
 
-`image-gc-high-threshold` may be set as a kubelet flag in /var/lib/kubelet/kubeadm-flags.env. Keep in mind that its value should be higher than `image-gc-low-threshold`, whose default value is 80%. An example of kubeadm-flags.env file:
+If non-container files occupy the disk space and the garbage collector cannot free enough space, this error may occur.
 
-```
+To resolve this issue:
+1. **Move `/var/lib/docker` to a separate disk** of reasonable size to free up space on the main disk.
+2. **Adjust `image-gc-high-threshold`** to a value lower than 100 minus `imagefs.available`. For example, setting it to 80% ensures that garbage collection starts earlier.
+
+The `image-gc-high-threshold` can be set as a kubelet flag in `/var/lib/kubelet/kubeadm-flags.env`. Ensure that its value is higher than `image-gc-low-threshold`, whose default is 80%. Here is an example of a `kubeadm-flags.env` file:
+```bash
 KUBELET_KUBEADM_ARGS="--cgroup-driver=systemd --network-plugin=cni --pod-infra-container-image=registry.k8s.io/pause:3.1 --kube-reserved cpu=200m,memory=256Mi --system-reserved cpu=200m,memory=512Mi --max-pods 250 --image-gc-high-threshold 80 --image-gc-low-threshold 70"
 ```
 
+### Recommendations
+- Regularly monitor disk space usage and garbage collection thresholds to prevent DiskPressure issues.
+- Consider moving Docker storage to a dedicated disk to avoid running out of space on the main disk.
+
+**Note**: Not applicable.
+
 ### Upgrade Procedure to v1.28.3 Fails on ETCD Step
```

Comment:
Looks like there is redundant entry in TOC for this one

Reply:
Removed the redundant entry